cli/src/main/java/bisq/cli/CurrencyFormat.java

```diff
@@ -50,6 +53,23 @@ public static String formatBsq(long sats) {
         return BSQ_FORMAT.format(BigDecimal.valueOf(sats).divide(BSQ_SATOSHI_DIVISOR));
     }
 
+    public static String formatTxFeeRate(TxFeeRateInfo txFeeRateInfo) {
+        String stdTxFeeRate = formatTxFeeRate(txFeeRateInfo.getStdTxFeeRate());
+        String customTxFeeRate = txFeeRateInfo.getCustomTxFeeRate() < 0
+                ? formatTxFeeRate(txFeeRateInfo.getCustomTxFeeRate())
+                : null;
```

Comment:
I don't understand what's going on here.

Reply:
To clear this up, I should probably add a `boolean useCustomWithdrawalTxFee` field to `TxFeeRateInfo`.  This would be set from the `Preference#useCustomWithdrawalTxFee` value.     (Remember that `Preferences` is not visible to the CLI, and these `*Info` proto wrappers are for passing the minimum data needed from server to CLI.)

This confusion is caused by not having that field.  When `txFeeRateInfo.getCustomTxFeeRate() == -1`, the custom tx fee rate preference is ignored in the CLI.  In core, it means the preference.useCustomWithdrawalTxFee == false.

In pseudo English, it means

```
String customTxFeeRate = (if are we using the fee service's tx fee rate)
                ? (format the custom tx fee rate)
                : null;

```

Adding a `boolean useCustomWithdrawalTxFee` field to `TxFeeRateInfo` would fix this.

Reply:
Fixed in commit faf030fbc5810e46a0d6fdab154b154ef39f72db.